index.html

```diff
@@ -9,16 +11,16 @@
     <aside>
       <h2>A Side Bar</h2>
     </aside>
-    <div class='main'>
+    <main class="main">
       <h2>The Main Content</h2>
       <div class='left'>
       	This is the left side
-      	<img src='https://jesse.sh/img/me.jpg'>
+      	<img src='https://www.jesse.sh/img/me.jpg'>
```

Comment:
It's actually missing an `alt=` tag. If you threw this `<img>` into a validator, it would complain that you don't have an `alt=` tag. This is primarily for accessibility reasons.